Mathlib/GroupTheory/GroupAction/FixingSubgroup.lean

```diff
@@ -36,6 +37,10 @@ TODO :
 
 * Treat semigroups ?
 
+* add `to_additive` for the various lemmas
+
+* `movingSubgroup`
```

Comment:
What is `movingSubgroup`?

Reply:
Ah, I see. It's the fixing subgroup of the complement. Are you still planning on adding this, or should this TODO be removed?

Reply:
No, I'm not going to add it for the same reason that `movedBy` was scrapped